globus_automate_client/flows_client.py

```diff
@@ -897,16 +961,16 @@ def flow_action_update(
             argument, that gets used to run the Flow operation. Otherwise the
             authorizer_callback defined for the FlowsClient will be used.
         """
+
         payload = {}
-        if run_managers is not None:
+        if run_managers:
             payload["run_managers"] = run_managers
-        if run_monitors is not None:
+        if run_monitors:
```

Comment:
I think this should be reverted. As the docstring says, an empty list actually has a particular meaning

Reply:
100% correct. I was on a kick to remove empty values. This needs to be undone.